startupscript/butane/devcontainer-failure-handler.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
set -o errexit
set -o nounset
set -o pipefail
set -o xtrace

can you add these 

and also a short paragraph about what this script do?